ocpi-endpoints-cpo-locations/src/main/scala/com/thenewmotion/ocpi/locations/LocationsRejectionHandler.scala

```diff
@@ -1,58 +0,0 @@
-package com.thenewmotion.ocpi.locations
-
-import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
-import akka.http.scaladsl.server.{AuthorizationFailedRejection, RejectionHandler}
-import akka.http.scaladsl.server.directives.{BasicDirectives, MiscDirectives}
-import akka.http.scaladsl.model._
-import akka.http.scaladsl.server._
-import StatusCodes._
-import Directives._
-import com.thenewmotion.ocpi.locations.LocationsError._
-import com.thenewmotion.ocpi.msgs.v2_1.CommonTypes.ErrorResp
-import com.thenewmotion.ocpi.msgs.v2_1.OcpiStatusCode.GenericClientFailure
-import org.joda.time.DateTime
-
-object LocationsRejectionHandler extends BasicDirectives with MiscDirectives with SprayJsonSupport {
-
-  import com.thenewmotion.ocpi.msgs.v2_1.OcpiJsonProtocol._
-
-  val DefaultErrorMsg = "An error occurred."
-  val Default = RejectionHandler.newBuilder().handle {
-    case AuthorizationFailedRejection =>
-      extractUri { uri =>
-        complete {
-          (Forbidden,
-            ErrorResp(
-              GenericClientFailure,
-              Some(s"The client is not authorized to access ${uri.toRelative}"),
-              DateTime.now()))
-        }
-      }
-
-    case LocationsErrorRejection(e@LocationNotFound(reason)) => complete {
-      ( NotFound,
```

Comment:
Are you sure about https://github.com/NewMotion/ocpi-endpoints/pull/7/commits/82568cd114db3419850c60e261b55b31893b6db5#r97692346 ?